.github/workflows/integration.yml

```diff
@@ -245,28 +292,62 @@ jobs:
-          mkdir -p etc/grid-security/certificates
+          mkdir -p etc/grid-security/
           echo "$HOSTCERT_BASE64" | base64 --decode > etc/grid-security/hostcert.pem
           echo "$HOSTKEY_BASE64" | base64 --decode > etc/grid-security/hostkey.pem
           chmod 440 etc/grid-security/hostcert.pem
           chmod 400 etc/grid-security/hostkey.pem
       - name: tests
-        env:
-          X509_CERT_DIR: /cvmfs/lhcb.cern.ch/etc/grid-security/certificates
-          X509_VOMS_DIR: /cvmfs/lhcb.cern.ch/etc/grid-security/vomsdir
-          DIRAC_VOMSES: /cvmfs/lhcb.cern.ch/etc/grid-security/vomses
         run: |
           cd Pilot
-          export VO_LHCB_SW_DIR=${GITHUB_WORKSPACE}/Pilot
           curl https://lhcbdirac.s3.cern.ch/Pilot3/LHCbPilotCommands.py -o LHCbPilotCommands.py
-          cp ../tests/CI/pilot_oldSchema.json pilot.json
+          cp ../tests/CI/${{ matrix.pilot_schema }} pilot.json
           sed -i "s/VAR_JENKINS_SITE/DIRAC.Jenkins.ch/g" pilot.json
-          sed -i "s/VAR_JENKINS_CE/jenkins-lhcb.cern.ch/g" pilot.json
+          sed -i "s/VAR_JENKINS_CE/${{ matrix.ce }}/g" pilot.json
+          sed -i "s/VAR_JENKINS_QUEUE/jenkins-queue_not_important/g" pilot.json
+          sed -i "s/VAR_DIRAC_VERSION/v11.0.20/g" pilot.json
+          sed -i "s#VAR_CS#https://lbcertifdirac70.cern.ch:9135/Configuration/Server#g" pilot.json
+          sed -i "s#VAR_USERDN#/DC=ch/DC=cern/OU=Organic Units/OU=Users/CN=alboyer/CN=819281/CN=Alexandre Franck Boyer#g" pilot.json
+          sed -i "s#VAR_USERDN_GRIDPP#${DIRACUSERDN_GRIDPP}#g" pilot.json
+          python dirac-pilot.py -M 1 -S DIRAC-Certification -N ${{ matrix.ce }} -Q jenkins-queue_not_important -n DIRAC.Jenkins.ch -e LHCb -l LHCb -E LHCbPilot --preinstalledEnvPrefix=/cvmfs/lhcb.cern.ch/lhcbdirac/ --cert --certLocation=${GITHUB_WORKSPACE}/Pilot/etc/grid-security --wnVO=LHCb -o cvmfsOnly --debug
+
+  ext-lhcb_integration_no_env_devel:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        pilot_schema:
+          - pilot_oldSchema.json
+          - pilot_newSchema.json
+
+    steps:
+      - uses: actions/checkout@v4
+      - uses: cvmfs-contrib/github-action-cvmfs@v3
+
+      - name: Test CernVM-FS
+        run: ls /cvmfs/lhcb.cern.ch
+
+      - name: Retrieve the secret and decode it to a file
+        env:
+          HOSTCERT_BASE64: ${{ secrets.HOSTCERT_BASE64 }}
+          HOSTKEY_BASE64: ${{ secrets.HOSTKEY_BASE64 }}
+        run: |
+          cd Pilot
+          mkdir -p etc/grid-security/
+          echo "$HOSTCERT_BASE64" | base64 --decode > etc/grid-security/hostcert.pem
+          echo "$HOSTKEY_BASE64" | base64 --decode > etc/grid-security/hostkey.pem
+          chmod 440 etc/grid-security/hostcert.pem
+          chmod 400 etc/grid-security/hostkey.pem
+      - name: tests
+        run: |
+          cd Pilot
+          curl https://gitlab.cern.ch/fstagni/LHCbPilot/-/raw/refactor_1/LHCbPilot/LHCbPilotCommands.py -o LHCbPilotCommands.py
```

Comment:
Note to myself: fix this once https://gitlab.cern.ch/lhcb-dirac/LHCbPilot/-/merge_requests/49 is merged (and add `master` branch too).